webapp/tabula_web.rb

```diff
@@ -77,6 +77,62 @@ def upload(file)
   return [job_batch, file_id]
 end
 
+
+def create_template(template_info)
+  puts template_info.inspect
+  template_name = template_info["name"] || "Unnamed Template #{Time.now.to_s}"
```

Comment:
I inlined it in the route where it's called. I think it's not really about manipulating the workspace, but instead about manipulating templates. https://github.com/tabulapdf/tabula/pull/711/commits/e9fd39474e0cdb19c096eca7308c37e50ecdd6ce